open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/dialog/MiniStyleTypePanel.java

```diff
@@ -0,0 +1,458 @@
+import io.opensphere.mantle.data.geom.style.VisualizationStyleParameter;
+import io.opensphere.mantle.data.geom.style.VisualizationStyleParameterChangeEvent;
+import io.opensphere.mantle.data.geom.style.VisualizationStyleParameterChangeListener;
+import io.opensphere.mantle.data.geom.style.VisualizationStyleRegistry.VisualizationStyleRegistryChangeListener;
+import io.opensphere.mantle.data.geom.style.impl.AbstractFeatureVisualizationStyle;
+import io.opensphere.mantle.data.geom.style.impl.VisualizationStyleRegistryChangeAdapter;
+import io.opensphere.mantle.util.MantleToolboxUtils;
+
+/**
+ * The Class MiniStyleTypePanel.
+ */
+public class MiniStyleTypePanel extends JPanel
+{
+    /** The unique identifier used for serialization operations. */
+    private static final long serialVersionUID = -3184666636503734481L;
+
+    /** The Constant FEATURE_TYPE_COLLAPSED_PREFIX. */
+    private static final String FEATURE_TYPE_COLLAPSED_PREFIX = "FeatureTypeCollapsed.";
+
+    /** Minus icon. */
+    private static final ImageIcon MINUS_ICON = new ImageIcon(MiniStylePanel.class.getResource("/images/minus.gif"));
+
+    /** Plus icon. */
+    private static final ImageIcon PLUS_ICON = new ImageIcon(MiniStylePanel.class.getResource("/images/plus.gif"));
+
+    /** The Current fvcp. */
+    private transient FeatureVisualizationControlPanel myCurrentFVCP;
+
+    /** The Destroyed. */
+    private boolean myDestroyed;
+
+    /** The currently selected data group. */
+    private final transient DataGroupInfo myDataGroup;
+
+    /** The currently selected datatype. */
+    private final transient DataTypeInfo myDataType;
+
+    /** The Expand collapse button. */
+    private JToggleButton myExpandCollapseButton;
+
+    /** The Feature class. */
+    private final transient Class<? extends VisualizationSupport> myFeatureClass;
+
+    /** The Is first. */
+    private final boolean myIsFirst;
+
+    /** The Mini style panel controller. */
+    private final transient MiniStyleTypePanelController myMiniStylePanelController;
+
+    /** The Registry style. */
+    private transient VisualizationStyle myRegistryStyle;
+
+    /** The Reset style button. */
+    private IconButton myResetStyleButton;
+
+    /** The Style select combo box. */
+    private final JComboBox<StyleNodeUserObject> myStyleSelectComboBox;
+
+    /** The Style select cb action listener. */
+    private final transient ActionListener myStyleSelectCBActionListener;
+
+    /** The Toolbox. */
+    private final transient Toolbox myToolbox;
+
+    /** The Style parameter change listener. */
+    private transient VisualizationStyleParameterChangeListener myVisStyleParameterChangeListener;
+
+    /** The Registry change listener. */
+    private transient VisualizationStyleRegistryChangeListener myVisStyleRegistryChangeListener;
+
+    /**
+     * Instantiates a new mini style type panel.
+     *
+     * @param tb the tb
```

Comment:
EX: shorthand parameter names & descriptions, unlike the other classes

Reply:
fixed